### DISCUSSION TITLE
希望加入文章pdf导出功能

### DISCUSSION THREAD

**Discussion by jadehh** [halo-dev/halo#6475] [Ideas]
### 你当前使用的版本

1.5.4

### 描述一下此特性

加入文章pdf导出功能

**Comment by ruibaby** [comment]
/triage needs-information

能更具体一点描述一下这个功能吗？

**Comment by jadehh** [comment]
类似于下面这种 https://github.com/alanshaw/markdown-pdf

**Comment by ruibaby** [comment]
此 issue 已过时，当前 Halo 2 已经支持插件机制，目前不考虑在核心中添加导出 PDF 的功能，以下为两种实现方案：

> 看看是否有社区的开发者愿意实现。

- 在主题的文章页面添加打印的功能，可以直接利用浏览器的 API 导出 PDF。
- 在插件中实现导出 PDF。

/close

[a comment by an automated account is not shown]

**Comment by ruibaby** [comment]
现在 Halo 2.x 已经有了一个支持导出 PDF 的插件。

https://github.com/ruibaby/plugin-content-tools/
